zrml/prediction-markets/src/benchmarks.rs

```diff
@@ -1323,6 +1323,38 @@ benchmarks! {
             CENT.saturated_into()
     )
 
+    manually_close_market {
+        let o in 1..63;
+
+        let range_start: MomentOf<T> = 100_000u64.saturated_into();
+        let range_end: MomentOf<T> = 1_000_000u64.saturated_into();
+
+        let (caller, market_id) = create_market_common::<T>(
+            MarketCreation::Permissionless,
+            MarketType::Categorical(T::MaxCategories::get()),
+            ScoringRule::Lmsr,
+            Some(MarketPeriod::Timestamp(range_start..range_end)),
+            Some(MarketDisputeMechanism::Court),
+        )?;
+
+        let now = 1_500_000u32;
+        assert!(range_end < now as u64);
+
+        let range_end_time_frame = Pallet::<T>::calculate_time_frame_of_moment(range_end);
+        for i in 0..o {
+            MarketIdsPerCloseTimeFrame::<T>::try_mutate(range_end_time_frame, |ids| {
+                ids.try_push((i).into())
+            }).unwrap();
+        }
```

Comment:
If you want exactly `o` entires in that vector, it needs to be this. Also changing from `i` to `i+1` because otherwise you'll have a double zero entry in the vector, which is inconsistent state.

```suggestion
        let range_end_time_frame = Pallet::<T>::calculate_time_frame_of_moment(range_end);
        for i in 1..o {
            MarketIdsPerCloseTimeFrame::<T>::try_mutate(range_end_time_frame, |ids| {
                ids.try_push((i + 1).into())
            }).unwrap();
        }
```